src/test/cls_queue/test_cls_queue.cc

```diff
@@ -62,20 +62,20 @@ static bool cmp_objs(cls_rgw_obj& obj1, cls_rgw_obj& obj2)
          (obj1.key == obj2.key) &&
          (obj1.loc == obj2.loc);
 }
-
+#if 1
 TEST(cls_queue, gc_queue_ops1)
 {
```

Comment:
@cbodley :should i rename the test folder also to cls_rgw_gc? and the test file name also to test_cls_rgw_gc.cc

Reply:
yes please, the cls_rgw_gc tests should move into their own subdir